User docs minor edit (#229)
AdaptiveMotorControlLab/CEBRA: docs/source/usage.rst

.. rubric:: Temperature :py:attr:`~.CEBRA.temperature`

> :py:attr:`~.CEBRA.temperature` has the largest effect on *visualization* of the embedding (see :py:doc:`cebra-figures/figures/ExtendedDataFigure2`). Hence, it is important that it is fitted to your specific data. A more smooth embedding will be achieved with a temperature set to 0.1, while 1.0 will be more "clustered".

Reviewer: ```suggestion
:py:attr:`~.CEBRA.temperature` has the largest effect on *visualization* of the embedding (see :py:doc:`cebra-figures/figures/ExtendedDataFigure2`). Hence, it is important that it is fitted to your specific data. Lower temperatures (e.g. around 0.1) will result in a more dispersed embedding, higher temperatures (larger than 1) will concentrate the embedding.
```

Reviewer: (sentence before mostly for discrete labels)

.. testoutput::

    CEBRA(batch_size=1024, learning_rate=0.001, max_iterations=10,
          model_architecture='offset10-model', temperature_model = 'constant', temperature = 0.1,

>           time_offsets=10)

Reviewer: this should be

```
CEBRA(batch_size=1024, learning_rate=0.001, max_iterations=10,
          model_architecture='offset10-model', time_offsets=10)
```

default args are not displayed

Displaying the temperature
""""""""""""""""""""""""""

> :py:attr:`~.CEBRA.temperature` has the largest effect on the visualization of the embedding. Hence it might be interesting to check its evolution when ``temperature_mode=auto``. We recommend only using `auto` if you have first explored the `constant` setting.

Reviewer: ```suggestion
:py:attr:`~.CEBRA.temperature` has the largest effect on the visualization of the embedding. Hence it might be interesting to check its evolution when ``temperature_mode=auto``. We recommend only using `auto` if you have first explored the `constant` setting. If you use the ``auto`` mode, please always check the time evolution of the temperature over time alongside the loss curve.
```

🧐 Below is a (non-exhaustive) list of actions you can try if your embedding looks different from what you were expecting.

#. Assess that your model `converged <https://machine-learning.paperspace.com/wiki/convergence>`_. For that, observe if the training loss stabilizes itself around the end of the training or still seems to be decreasing. Refer to `Visualize the training loss`_ for more details on how to display the training loss.

> #. Increase the number of iterations. It typically should be at least 10,000.

Reviewer: ```suggestion
#. Increase the number of iterations. It typically should be at least 10,000. On small datasets, it can make sense to stop training earlier to avoid overfitting effects.
```
